website/blog/2020-01-10-cobalt.md

```diff
@@ -0,0 +1,718 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.0 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+After a long while of silence and with loads of new features we are excited to
+announce the release of Metals v0.8.0 codename `Cobalt`. This release includes a
+huge number of new features and improvements.
+
+Metals is a language server for Scala that works with VS Code, Atom, Vim,
+Sublime Text and Emacs. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+along with contributors from the community.
+
+In this release we merged 201 PRs and closed 24 issues, full details:
+https://github.com/scalameta/metals/milestone/9?closed=1
+
+## TL;DR
+
+Check out the website and give Metals a try: https://scalameta.org/metals/
+
+- goto implementation
+- rename symbol
+- support for Pants build tool
+- run and debug for tests and main methods
+- breakpoints
+- improvements for Bloop integration
+- first code action - import missing symbol
+- worksheet support
+- Add completion to add all abstract members To mention
+
+## Goto implementation
+
+TODO
+
+## Rename symbol
```

Comment:
Suggestion: have a "Refactors" section, which explains we've started implementing refactors. "Rename" is the first huge one (which is why it has special treatment in LSP) and we're now starting to tackle other refactors via code actions. Import symbol is the first one.

I think from a user perspective those features fall under the same hat.

Reply:
I also like the idea of breaking them up into a couple sections. I think that lends to better readability.